tools/topology/topology2/include/platform/intel/dmic-generic.conf

```diff
@@ -0,0 +1,196 @@
+<include/dais/dmic.conf>
+<include/dais/pdm_config.conf>
+# This file should be included from other top-level topology files. So do not include other
+# conf's in this file.
+
+#
+# Arguments to be passed when compiling this conf file. All of them come with default values.
+# These can be modified by passing the argument values with alsatplg as follows:
+# alsatplg -D DMIC0_ID=4,DMIC1_ID=5,DMIC_CH_COUNT=4 -p -c dmic_generic.conf -o output.conf
+#
+
+Define {
+	# Link ID for DMIC 480KHz in the machine driver
+	DMIC0_ID		6
```

Comment:
this seems to be specific to HDAudio, I don't think ID works in general? Should this be hda-dmic.conf?